view/frontend/templates/script/sentry.phtml

```diff
@@ -16,6 +16,32 @@ if (!$block->canUseScriptTag($block->getNameInLayout())) {
         dsn: '<?= $block->escapeUrl(trim($block->getDSN())) ?>',
         release: '<?= $block->escapeHtml(trim($block->getVersion())) ?>',
         environment: '<?= $block->escapeHtml(trim($block->getEnvironment())) ?>',
+        <?php if ($block->stripStaticContentVersion() || $block->stripStoreCode()): ?>
+        beforeSend: function(event) {
+            event.exception.values.map(function (v,i) {
```

Comment:
Would you be able to expand the variable names in the JS?
For instance naming `v` `exception`
and is `i` the index in the list?
`i` seems unused and is not required to be there if it is not used 
https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map